bsp/m5stack_core_s3/idf_component.yml

```diff
@@ -9,7 +9,7 @@ tags:
   - bsp
 
 dependencies:
-  idf: ">=5.0"
+  idf: ">=5.4" # I2C Driver-NG is supported in esp32-camera only from IDFv5.4
```

Comment:
@igrr FYI, we must drop IDF v5.0-5.3 support because of this https://github.com/espressif/esp32-camera/blob/master/CMakeLists.txt#L87